pkg/indexmanagement/reconcile.go

```diff
@@ -47,7 +47,7 @@ var (
 	millisPerWeek   = uint64(millisPerDay * 7)
 
 	//fullExecMode 0777
-	fullExecMode = utils.GetInt32(int32(511))
+	fullExecMode = utils.GetInt32(int32(0o777))
```

Comment:
The octal syntax is supported in golang 1.13 right? I can't recall.

Reply:
I'd very very surprised if it wasn't supported in Go 1.0 :)

Reply:
Wait, you're right, let's fix this

Reply:
Yes, have a look here: https://golang.org/doc/go1.13#language
However, we shouldn't have a problem. I updated the builders to 1.13 during migration to go modules.

Reply:
Right, but I've changed it to just a 0 prefix, that's been there since day one, just in case. There're still remnants of pre-1.13 Go in some places, like https://github.com/openshift/cluster-logging-operator/blob/5db9f23063de2892078cfe262cf4aaa81335b5e2/Dockerfile#L1